### PR TITLE
Adds validation steps

### PR DESCRIPTION
**Pull request by brianseek**
Added validation steps to frontend ticket template for check-in-experience.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/check-in-experience-development-frontend.md [hunk 1]
```diff
@@ -39,3 +39,10 @@ As a <user>, I want to <do something> so that I can <accomplish a goal>
 - [ ] Prod validation:
   - [ ] not necessary
   - [ ] scheduled
+
+## Validation Steps
```
Comment (brdunfield) [inline .github/ISSUE_TEMPLATE/check-in-experience-development-frontend.md:43]: Could we maybe re-word this to be Code or Technical validation, to distinguish from our UX validation process?

File: .github/ISSUE_TEMPLATE/check-in-experience-development-frontend.md [hunk 2]
```diff
@@ -39,3 +39,10 @@ As a <user>, I want to <do something> so that I can <accomplish a goal>
...
+- [ ] Read ticket details
```
Comment (acrollet) [inline .github/ISSUE_TEMPLATE/check-in-experience-development-frontend.md:44]:
```suggestion
- [ ] Read ticket details and verify that ACs are met
```

**Review by brdunfield — APPROVED** [review]
👍

**Review by acrollet — APPROVED** [review]
🐙

**Pull request merged**